src/styles/utils/visually-hidden.scss

```diff
@@ -0,0 +1,11 @@
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
+}
```

Comment:
why did you use these styles? just try don't use !important

Reply:
Just for that class , so screen readers can still see the h1 , like misha did.